webapp/src/components/DynamicButtonDataTable.vue

```diff
@@ -24,44 +24,64 @@
       </button>
       <button
         v-if="dataType === 'collections'"
-        class="btn btn-default"
+        class="btn btn-default ml-2"
         @click="$emit('open-create-collection-modal')"
       >
         Create new collection
       </button>
       <button
         v-if="dataType === 'startingMaterials' && editableInventory"
-        class="btn btn-default"
+        class="btn btn-default ml-2"
         @click="$emit('open-create-item-modal')"
       >
         Add a starting material
       </button>
       <button
         v-if="dataType === 'equipment'"
-        class="btn btn-default"
+        class="btn btn-default ml-2"
         @click="$emit('open-create-equipment-modal')"
       >
         Add an item
       </button>
     </div>
     <div class="button-right d-flex">
-      <button
-        v-if="itemsSelected.length > 0 && dataType != 'collections'"
-        class="btn btn-default"
-        :disabled="itemsSelected.length === 0"
-        @click="$emit('open-add-to-collection-modal')"
-      >
-        Add to collection
-      </button>
-      <button
-        v-if="itemsSelected.length > 0"
-        class="btn btn-default ml-2"
-        data-testid="delete-selected-button"
-        :disabled="itemsSelected.length === 0"
-        @click="confirmDeletion"
-      >
-        Delete selected
-      </button>
+      <div class="dropdown">
+        <button
+          data-testid="selected-dropdown"
+          class="btn btn-default dropdown-toggle"
+          type="button"
+          data-toggle="dropdown"
+          aria-haspopup="true"
+          aria-expanded="false"
+          :disabled="itemsSelected.length === 0"
+          @click="isSelectedDropdownVisible = !isSelectedDropdownVisible"
+        >
+          {{ itemsSelected.length > 0 ? `${itemsSelected.length} selected ` : "Selected ... " }}
```

Comment:
```suggestion
          {{ itemsSelected.length > 0 ? `${itemsSelected.length} selected... ` : "Selected ... " }}
```

I like this working, but think we should keeping the elippses on when items are selected so it will read "n selected...", showing that you can click on that button for more options.